tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/Lambda/LambdaCommon.cs

```diff
@@ -18,8 +18,9 @@ namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AWS.Lambda;
 
 internal abstract class LambdaCommon
 {
-    private const string PlaceholderServiceName = "placeholder-service";
-    private const string PlaceholderOperationName = "placeholder-operation";
+    // Name of the placeholder span sent which will be filtered out by the Lambda Extension,
+    // required so we can get correct span metadata, tags, and more.
+    private const string InvocationSpanResource = "dd-tracer-serverless-span";
```

Comment:
This field name seems wrong. `...SpanResource`, but it's used for operation name (aka span name) and service name, not the resource name.

Reply:
I don't think we're setting the resource name at all on this span. Should we?

Reply:
You're right, I should rename it!

And I don't think adding the span resource matters because it's going to be dropped anyway, the only data we care about are the meta and meta metrics when sending it